frontend/src/lib-components/messages/MessageReplyEditor.tsx

```diff
@@ -102,6 +102,7 @@ function MessageReplyEditor<T, R>({
           onChange={(value) => onUpdateContent(value)}
           data-qa="message-reply-content"
           autoFocus
+          autoFocusDelayMs={200}
```

Comment:
Tästä auto focus delaystä tulee ruudunlukijan käyttäjälle semmoinen huononnus, että focus on tuon 200ms dokumentin root-tasolla ennen kuin se siirtyy textareaan. Tämä johtuu siitä, että fokusoitu elementti (Vastaa viestiin) häviää DOMista. Kun fokus siirtyy dokumentin root-tasolle, ruudunlukija alkaa lukemaan sivun titleä "Varhaiskasvatus" josta se kerkeää lukemaan vain osan ennen kuin focus siirtyy -> textarea.

Keksisiköhän tähän jonkun sellaisen ratkaisun, että ruudunlukijan käyttäjille ei tulisi tämmöistä huononnusta?

Reply:
Yeah, your understanding is correct about the purpose of the delay. In case the textarea is outside of the screen when it is focused, it will automatically jump to the middle of the screen. Only after that it will start to scroll smoothly. But because of the jump earlier, it looks like the scrolling happens from the wrong direction.

One option could be just to get rid of this specific commit. Or somehow keep the button around until the delayed focus happens.

Reply:
Fixed by adding a parameter that allows focusing without scrolling